apis/imdb.py

```diff
@@ -14,25 +26,30 @@ def fetch_imdb(url: str) -> Dict:
     ).json()
 
     if len(IMDB_data["movie_results"]) > 0:

```

Comment:
is there a better check we can implement here? What if we checked if the first object had a certain set of keys? This is better since imdb might send back 1 value in the array but it may be invalid. Then the following code would break anyways. Also consider throwing if the data doesnt have the proper keys.

Reply:
How about:
`if IMDB_data['movie_results']:`

the API would actually populate only one key for one product (either movie, tv series, or person) and keep the rest empty. Not sure if an invalid value may occur but I can add more checks when refining the data.